uiServices/src/testCaseServices_test.go

```diff
@@ -355,3 +355,63 @@ func TestGetAllCasesNoHeader(t *testing.T) {
 }
 
 
+
```

Comment:
This is personal preference but I would suggest you try and rewrite the tests as **Table driven test**. See example here: https://medium.com/@matryer/5-simple-tips-and-tricks-for-writing-unit-tests-in-golang-619653f90742
This would be a good exercise.

Reply:
I've made changes based on your feedback. I have not implemented Test driven tables as time is limited and we only have a week left to finish stories for this project so that's my priority over learning new techniques but thanks for showing it to me, seems useful!

Reply:
Yeah I will add another story for the tabled tests. If we get time.

Reply:
XAPTC-68  has been created to handle this.